BirthdayBot/Cogs/BirthdayChecker.py

```diff
@@ -73,3 +78,56 @@ def generateRandomMessage(self) -> dict:
             }
 
             return bdayMessage
+
+
+class BirthdayCommands(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.hybrid_command(
+        name="today",
+        description="Displays everyoen with birthdays for the day.",
+    )
+    async def today(self, ctx):
+        guildId = ctx.message.guild
+        todayBdays = BirthdayChecker.getAllBirthdays(guildId)
+        month = datetime.today().month
+        day = datetime.today().day
+        numBdays = 1
+        embed = discord.Embed(
+            title=f"Todays Birthday's - {month}/{day}",
+            description="List of people with birthdays today:",
+            color=discord.Color.red(),
+        )
+        await ctx.send(embed=embed)
+        for birthdays in todayBdays:
+            userAge = UserAgeInfo.getUserAge(birthdays.Birthday)
+            user = await ctx.guild.query_members(user_ids=[int(birthdays.discord_ID)])
+            user = user[0]
+            embed2 = discord.Embed(
+                title=f"{birthdays.username}",
+                description=f"is {userAge} today!",
+                color=discord.Color.red(),
+            )
+            embed2.set_image(url=user.avatar.url)
+            embed2.set_footer(text=f"{numBdays}/{len(todayBdays)}")
+            numBdays += 1
+            await ctx.send(embed=embed2)
+
+    @commands.hybrid_command(
+        name="tomorrow",
+        description="Displays users birthdays for tomorrow.",
+    )
+    async def tomorrow(self, ctx):
+        await ctx.send("coming soon...")
+
+    @commands.hybrid_command(
+        name="thismonth",
+        description="Displays users birthdays for the month.",
+    )
+    async def thismonth(self, ctx):
+        await ctx.send("coming soon...")
```

Comment:
Should probably make a task/issue to get these implemented at some point (so we don't forget)